msi-ec.c

```diff
@@ -509,6 +782,8 @@ static DEVICE_ATTR_RW(webcam);
 static DEVICE_ATTR_RW(fn_key);
 static DEVICE_ATTR_RW(win_key);
 static DEVICE_ATTR_RW(battery_mode);
+static DEVICE_ATTR_RO(lid_status);
+static DEVICE_ATTR_RO(ac_connected);
```

Comment:
5. Please, document them in the README file under the usage section

Reply:
Yep, completely forgot about README

Reply:
Or do we even need these? Just like battery capacity and status, they already have generic implementations

Reply:
It's true, you can get rid them.

Reply:
Done